documentation/src/pages/recipes/data/recipes/refactor-function.yaml

```diff
@@ -0,0 +1,35 @@
+version: "1.0.0"
+title: "Refactor Function"
```

Comment:
please add the 'author' section as well, which includes your github username, for example:

```yaml
author:
  contact: better-boy
```